backend/resources/gpml/config.edn

```diff
@@ -84,13 +84,45 @@
-                                                                 #ig/ref :gpml.auth/approved-user
-                                                                 #ig/ref :gpml.auth/admin-required-middleware]}
+                                                                 #ig/ref :gpml.auth/approved-user]}
                                          ["/get-reviewers"
                                           {:get {:summary "Get list of reviewers"
+                                                 :middleware [#ig/ref :gpml.auth/admin-required-middleware]
+                                                 :swagger {:tags ["review"]
+                                                           :security [{:id_token []}]}
+                                                 :handler #ig/ref :gpml.handler.review/get-reviewers}}]
+                                         ["/{topic-type}/{topic-id}"
+                                          {:get {:summary "Get review for a submission"
+                                                 :middleware [#ig/ref :gpml.auth/admin-required-middleware]
                                                  :swagger {:tags ["review"]
                                                            :security [{:id_token []}]}
-                                                 :handler #ig/ref :gpml.handler.review/get-reviewers}}]]
+                                                 :parameters {:path [:map
+                                                                     [:topic-type #ig/ref :gpml.handler.detail/topics]
+                                                                     [:topic-id int?]]}
+                                                 :handler #ig/ref :gpml.handler.review/get-review}}
+                                          {:patch {:middleware [#ig/ref :gpml.auth/reviewer-required-middleware]
+                                                   :summary "Update review for a submission"
+                                                   :swagger {:tags ["review"]
+                                                             :security [{:id_token []}]}
+                                                   :parameters {:path [:map
+                                                                       [:topic-type #ig/ref :gpml.handler.detail/topics]
+                                                                       [:topic-id int?]]
+                                                                :body [:map
+                                                                       [:review-comment string?]
+                                                                       [:review-status #ig/ref :gpml.handler.review/review-status]]}
+                                                   :handler #ig/ref :gpml.handler.review/update-review}}]
+                                         ["/assign-reviewer/{topic-type}/{topic-id}"
```

Comment:
Thinking in taking out from `/submission` and use also `/reviewer`
`POST /reviewer/topic-type/topic-id` 
BTW: why here (API) the first arg is called `topic-type` but in backend and db is called `topic-name` ... should we unify names here?

Reply:
Agree on unifying names for `topic-type` and `topic-name`.  

The `assign-reviewer` is currently in `/review` and not `/submission`. So, I'm not sure what your entire suggestion here is. :)